### PR TITLE
✨ allow users to select pre-existing intros

### LINKED ISSUE #69: Consider allowing reuse of already uploaded songs
We would want to add an option to the /settings command with autocomplete to allow users to pick from pre-existing songs.

### PR DESCRIPTION
**Pull request by dudeofawesome**
Closes #69

### CODE REVIEW

File: src/bot/commands/settings.command.ts [hunk 1]
```diff
@@ -106,24 +138,67 @@ export class SettingsCommand {
...
+          hash: dto.intro_song_hash,
+          public: true,
+        });
+        if (intro_song_ent == null) {
+          messages.push('Could not find the requested song.');
+        } else if (
+          intro_song_ent.duration_ms >
+          secondsToMilliseconds(guild_ent.intro.max_duration_s)
+        ) {
+          messages.push('Selected song it too long.');
```
Comment (EdgarSaldivar) [inline src/bot/commands/settings.command.ts]: it -> is

File: src/bot/commands/settings.command.ts [hunk 2]
```diff
@@ -156,19 +231,19 @@ export class SettingsCommand {
...
+      if (dto.clear_intro_song) {
+        await this.virgins.nativeUpdate(
+          { id: target_user_snowflake },
+          { intro_song: null },
+        );
+
+        messages.push(
+          `${pascal_spaces(
+            target_user_name,
+          )} intro song has been reset to default.`,
```
Comment (EdgarSaldivar) [inline src/bot/commands/settings.command.ts:244]: should this be capitalize 'intro' ?

**Review by EdgarSaldivar — COMMENTED** [review]
I found something :D

File: src/bot/commands/settings.command.ts (hunk 2 above)
Comment (dudeofawesome) [reply src/bot/commands/settings.command.ts:244]: This turns in to "Your intro song has been reset to default."

**Pull request merged**